### ISSUE TITLE
Support for F# script files

### ISSUE THREAD

**Issue by lamg** [nathanjcollins/zed-fsharp#4]
F# code can appear in files with the following extensions

- `.fs`: files in a project
- `.fsx`: scripts
- `.fsi`: module interface files
- `.fsl`: FsLex files
- `.fsy`: FsYacc files

Currently only `.fs` files are supported, however `.fsx` seem easy to support since they are almost syntactically equivalent to `.fs` files, with the only addition of `#r <string>` directives. `#r "nuget: OpenAI, 2.0.0"` for example.

**Comment by NatElkins** [comment]
@nathanjcollins Can this issue be closed?  Based on https://github.com/nathanjcollins/zed-fsharp/pull/5.

**Comment by nathanjcollins** [comment]
> [@nathanjcollins](https://github.com/nathanjcollins) Can this issue be closed? Based on [#5](https://github.com/nathanjcollins/zed-fsharp/pull/5).

@NatElkins yep I think I'll close this for now, I don't think I'm planning support for those filetypes
